### ISSUE TITLE
read_data missing

### ISSUE THREAD

**Issue by bernstei** [ACEsuit/ACEpotentials.jl#59]
I'm running the workflow's ace fit test, and I'm getting the following error
```
ERROR: LoadError: UndefVarError: read_data not defined
Stacktrace:
 [1] get_num_observations(d::Dict{String, String})
   @ Main ~/.julia/packages/ACE1pack/rq4n0/scripts/ace_fit.jl:21
 [2] save_dry_run_info(fit_params::Dict{String, Any})
   @ Main ~/.julia/packages/ACE1pack/rq4n0/scripts/ace_fit.jl:31
 [3] top-level scope
   @ ~/.julia/packages/ACE1pack/rq4n0/scripts/ace_fit.jl:56
in expression starting at /home/cluster2/bernstei/.julia/packages/ACE1pack/rq4n0/scripts/ace_fit.jl:55
```
I've updated the packages in julia, and it's reporting
```
      Status `~/.julia/environments/v1.7/Project.toml`
  [e3f9bc04] ACE1 v0.9.2
  [8c4e8d19] ACE1pack v0.2.0
  [51974c44] ASE v0.5.4
  [c7e460c6] ArgParse v1.1.4
  [945c410c] JuLIP v0.11.5
```

The actual run line is `julia /home/cluster2/bernstei/.julia/packages/ACE1pack/rq4n0/scripts/ace_fit.jl --params run_dir/fit_params_ACE.B_test.json > run_dir/ACE.B_test.stdout 2> run_dir/ACE.B_test.stderr`, but since this error appears to be internal to `ace_fit.jl`, I'm not sure those details matters.

@gelzinyte said she thinks the fix just involves updating the julia package's requested github release, because the bug has been fixed there.

**Comment by wcwitt** [comment]
Ack, this is probably my fault. We tagged the first ACE1pack (v0.2) associated with ACEfit, which has some breaking changes (documentation imminent, but I'm on holiday this week), and it looks like you're hitting those. The fastest solution is probably to restrict ACE1pack to v0.0 for now.

**Comment by cortner** [comment]
Please don't pin it manually but instead specify the version bounds in the `Project.toml` in your project folder (not in ACE1pack!!). The following should work: 
```toml
[compat] 
ACE1pack = "< 0.2.0"
```

**Comment by gelzinyte** [comment]
I have fixed this particular issue in the latest commit, so it should work in `dev` mode. Or an older ACE1pack version should be good too (as @wcwitt suggested). But long-term these changes will get incorporated with the next version tag, not sure if the last bug fix is enough or we should wait for docs, etc. A unit test for `ace_fit.jl` will prevent bugs like this in the future - I have opened issue #60 as a reminder.
